apps/web-giddh/src/app/shared/eWayBill/create/e-way-bill-create-component.ts

```
@@ -152,53 +84,38 @@ export class EWayBillCreateComponent implements OnInit, OnDestroy {
-        this.selectedInvoices = this.invoiceService.getSelectedInvoicesList;
 
-        this.transporterListDetails$.subscribe(op => {
-            this.transporterListDetails = op;
+        this.componentStore.transporterListDetails$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
+            this.transporterListDetails = response;
         })
-        this.store.pipe(select(state => state.ewaybillstate.TransporterList), takeUntil(this.destroyed$)).subscribe(state => {
-            if (state && state.length) {
+
+        this.componentStore.transporterList$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
+            if (response && response.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)